tests/weight_converter/torch/test_torchscript.py

```diff
@@ -1,8 +1,7 @@
-from pathlib import Path
 from bioimageio.spec import export_package
 
 
 # TODO only run this test if we have pytorch
 def test_torchscript_converter(unet2d_nuclei_broad_model_url):
-    spec_path = export_package(unet2d_nuclei_broad_model_url, Path("./data"))
```

Comment:
it doesn't make sense to specify a root path here, the root_path is meant only to resolve relative paths in a given rdf (if the file is given as raw node or 'content dict'/'yaml.load(rdf_file)')

the returned path is in the BioImage.IO cache folder (in tmp)... do we need to add an output path argument?

Reply:
> it doesn't make sense to specify a root path here, the root_path is meant only to resolve relative paths in a given rdf (if the file is given as raw node or 'content dict'/'yaml.load(rdf_file)')

Ok, please add that to the documentation; and I think you could even raise a `ValueError` for invalid combination, i.e. url + root path, with a meaningful error message.

> the returned path is in the BioImage.IO cache folder (in tmp)

that's good here, as we just want to use the data for the tests.

> do we need to add an output path argument?

yes, I think that makes sense, but it should be optional (and if not given return the cache path)

Reply:
> Ok, please add that to the documentation; and I think you could even raise a ValueError for invalid combination, i.e. url + root path, with a meaningful error message.

new issue: https://github.com/bioimage-io/spec-bioimage-io/issues/202

> yes, I think that makes sense, but it should be optional (and if not given return the cache path)

new issue: https://github.com/bioimage-io/spec-bioimage-io/issues/203